required/amsmath/amsmath-2018-12-01.sty

```diff
@@ -166,9 +166,11 @@ For additional information on amsmath, use the \lq ?\rq\space option.%
 \let\negthinspace\!
 \renewcommand{\:}{\tmspace+\medmuskip{.2222em}}
 \let\medspace\:
+\let\negmedspace\@undefined % allow rollback (gh/1770)
 \newcommand{\negmedspace}{\tmspace-\medmuskip{.2222em}}
 \renewcommand{\;}{\tmspace+\thickmuskip{.2777em}}
 \let\thickspace\;
+\let\negthickspace\@undefined % allow rollback (gh/1770)
```

Comment:
perhaps it would be clearer if all these are moved up front with a comment there, or, given that the change in lines 2680 can't really be move just add at the top off the file something like
```
%%
%% Minor adjustments for the original file to allow rollback -- search for (gh/1770)
%% 
```

Reply:
done